DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -343,14 +349,18 @@ def verify_dcr_exists(self):
             return False
         return True
 
-    def verify_dcr_content_has_cef_stream(self):
+    def verify_dcr_content_has_stream(self):
         """
         Verifying there is a DCR on the machine for forwarding cef data
         """
-        command_name = "verify_DCR_content_has_CEF_stream"
+        command_name = "verify_DCR_content_has_stream"
+        if STREAM_SCENARIO == 'ftd':
+            self.STREAM_SCENARIO = 'asa'
```

Comment:
Do not use the same naming, use internal name e.g. _stream_scenario, also I think there is no need for self.

Reply:
Done.